external/os-autoinst-common/tools/update-deps

```diff
@@ -70,7 +70,7 @@ sub update_dockerfile {
     my $run = <<"EOM";
 # This part is autogenerated by $scriptname from $yamlfile
 RUN zypper in -y -C \\
-$dep   && zypper clean
+$dep   && true
```

Comment:
Looks like you're reverting your own commit https://github.com/os-autoinst/os-autoinst/commit/2cbac3be6ea9e0ed1d9e3e9eb0c9c343a82fa7d4#diff-9adfb2f70d4edf51600e7f7fb1825cc5bbc55cb8863d7676c542de85b3560430 here.

Reply:
hm, right. I should try to fix that by pushing to the common repo.

Reply:
Accepted the PR.

Reply:
well now it's too late because the external repo has changed too.
normally a `git subrepo pull -M rebase external/os-autoinst-common` should work, but I don't know why it doesn't here.
It doesn't even work in openQA where there are no conflicting changes.

Reply:
I will learn with time :) However now after https://github.com/os-autoinst/os-autoinst-common/pull/8 merged and doing another pull the diff here looks better